pkg/compiler/return_test.go

```diff
@@ -110,7 +111,7 @@ func TestNamedReturn(t *testing.T) {
 	runCase := func(ret string, result ...interface{}) func(t *testing.T) {
 		return func(t *testing.T) {
 			src := fmt.Sprintf(src, ret)
-			v := vmAndCompile(t, src)
+			v := vmAndCompile(t, src, &compiler.Options{NoMethodsCheck: true})
```

Comment:
Why don't you change the test instead? It can and should use local function with named return values.

Reply:
Fixed in a separate commit. Don't we want to keep "--no-methods" option?

Reply:
I don't want to add options to the compiler unless there is no way to avoid that. We better be as far away from `man gcc` as possible with our compiler.

Reply:
Fixed.